pyproject.toml

```diff
@@ -39,6 +39,7 @@ profile = "black"
 
 [tool.poetry.scripts]
 unblob = "unblob.cli:main"
+unromfs = "unblob.extractors.unromfs:main"
```

Comment:
Should we add the extractors directory to the coverage measurement 3 lines below? Hopefully subprocess coverage is tracked well since the parallelization.

Reply:
It's already covered by `--cov=unblob` since unromfs is in `unblob/extractors`.

Pytest coverage output:

```
---------- coverage: platform linux, python 3.8.10-final-0 -----------
Name                                             Stmts   Miss Branch BrPart  Cover
----------------------------------------------------------------------------------
unblob/extractors/__init__.py                        0      0      0      0   100%
unblob/extractors/unromfs.py                       269     39     91     14    81%
```